src/credman.c

```diff
@@ -249,6 +250,15 @@ credman_parse_rk(const cbor_item_t *key, const cbor_item_t *val, void *arg)
 			return (-1);
 		cred->type = cred->attcred.type; /* XXX */
 		return (0);
+	case 10:
+		if (cbor_decode_uint64(val, &prot) < 0)
+			return (-1);
+		if (prot != FIDO_CRED_PROT_UV_OPTIONAL &&
+		    prot != FIDO_CRED_PROT_UV_OPTIONAL_WITH_ID &&
+		    prot != FIDO_CRED_PROT_UV_REQUIRED)
+			return (-1);
+		cred->ext.prot = (int)prot;
```

Comment:
Would it make sense to use fido_cred_set_prot() here?

Reply:
Agreed and force-pushed.